addons/website/views/snippets/s_social_media.xml

```diff
@@ -2,7 +2,7 @@
 <odoo>
 
 <template id="s_social_media" name="Social Media">
-    <div class="s_social_media text-start o_not_editable" contenteditable="false">
+    <div class="s_social_media text-start o_not_editable" contenteditable="false" data-vxml="001">
```

Comment:
But this one has no xml changes so it doesn't need the vxml trick, am I missing something ?

Reply:
I'm likely the one missing something. Does modifying `website_templates.xml` not count as XML changes ?

Reply:
In any case, since we add the class on _all_ social media icons (and thus do that with xml changes) this will be required.